src/net_processing.cpp

```diff
@@ -295,7 +295,7 @@ struct Peer {
         std::atomic<std::chrono::seconds> m_last_mempool_req{0s};
         /** The next time after which we will send an `inv` message containing
          *  transaction announcements to this peer. */
-        std::chrono::microseconds m_next_inv_send_time GUARDED_BY(NetEventsInterface::g_msgproc_mutex){0};
+        std::chrono::microseconds m_next_inv_send_time GUARDED_BY(m_tx_inventory_mutex){0};
```

Comment:
in 845e3a34c49abcc634b5a10ccdd6b10fb4fcf449, noting to myself that this change is because `m_next_inv_send_time` can now be accessed from the scheduler thread in `ReattemptInitialBroadcast`.